rules/windows/defense_evasion_from_unusual_directory.toml

```
@@ -102,6 +102,7 @@ This rule identifies processes that are executed from suspicious default Windows
 - Determine the initial vector abused by the attacker and take action to prevent reinfection through the same vector.
 - Using the incident response data, update logging and audit policies to improve the mean time to detect (MTTD) and the mean time to respond (MTTR).
 """
+references = ["https://www.elastic.co/security-labs/invisible-miners-unveiling-ghostengine"]
```

Siestagraph executes from `Windows\\Tasks` covered in this rule : 

```suggestion
references = [
"https://www.elastic.co/security-labs/invisible-miners-unveiling-ghostengine", 
"https://www.elastic.co/security-labs/siestagraph-new-implant-uncovered-in-asean-member-foreign-ministry"
]
```